includes/widgets/image-carousel.php

```diff
@@ -71,20 +71,6 @@ public function get_keywords() {
 		return [ 'image', 'photo', 'visual', 'carousel', 'slider' ];
 	}
 
-	/**
-	 * Retrieve the list of scripts the image carousel widget depended on.
-	 *
-	 * Used to set scripts dependencies required to run the widget.
-	 *
-	 * @since 1.3.0
-	 * @access public
-	 *
-	 * @return array Widget scripts dependencies.
-	 */
-	public function get_script_depends() {
```

Comment:
Return this method back, and change the dependency to 'swiper'

Reply:
Are you sure we need this? None of the other Swiper carousels call the `get_script_depends` method with `'swiper'`.